internal/libvirt/libvirt.go

```diff
@@ -10,12 +10,12 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/digitalocean/go-libvirt"
+	golibvirt "github.com/digitalocean/go-libvirt"
```

Comment:
Why import aliasing for this? If it is necessary 
```suggestion
	libvirt "github.com/digitalocean/go-libvirt"
```

Reply:
I use golibvirt to avoid an import shadow